runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -1671,9 +1671,6 @@ ZZ  Check if it's a private method
     LR      r1,r2
     NILL    r1,J9TR_J9_VTABLE_INDEX_DIRECT_METHOD_FLAG
     JNZ     L_privateMethod
-ZZ  Skip patching if vft offset is smaller than -32768
```

Comment:
I forgot to check into the other path which this code is prepared to patch. It seems to be handling 2 case, one being the "LG, LHI" case and the other "A[G]HI, L[G]HI". I suspect that that other path also needs to use 32bit immediate instructions as well. So I think more work is need :-(

Reply:
This code was used to patch the sequence of code that would be generated when the displacement was unable to fit in 20bits. Testing suggests that the displacement is always going to fit in 20bits. Also added an assert to catch the problem should the assumption be incorrect.